polkadot/cli/src/cli.rs

```diff
@@ -150,7 +150,7 @@ pub struct RunCmd {
 	///
 	///**Dangerous!** This is an experimental feature and should not be used in production, unless
```

Comment:
If we enable by default, we should delete the comment. Polkadot relay chain nodes do not live a dangerous life :D

Reply:
Done.